### ISSUE TITLE
Is SB + decessive error possible?

### ISSUE THREAD

**Issue by AloisSeckar** [AloisSeckar/WBSC-Scoring#187]
Related to #186

Is it actually possible to have SB and then a decessive erorr?

![image](https://github.com/AloisSeckar/WBSC-Scoring/assets/7399922/862b0920-dcfd-43a7-a291-db1f6f12f627)

**Comment by lucafano04** [comment]
nope, not on the same "Ball" this is not possible, the error should me "small"

**Comment by AloisSeckar** [comment]
Answered by Dennis Duin / Linda Steijger from WBSC Europe:

_When a runner is attempting to steal and the catcher, after receiving the pitch, makes a wild throw to prevent the stolen base, we score a stolen base. In case the throw is muffed and the runner can take the opportunity to advance an extra base, it is an extra base error, not a decisive error._

--

Therefore, inserting such play must be restricted by validation - issue #277
